openassessment/xblock/static/js/src/lms/oa_grade.js

```diff
@@ -20,17 +22,31 @@ OpenAssessment.GradeView.prototype = {
         var view = this;
         var baseView = this.baseView;
         var stepID = ".step--grade";
+        view.isRendering = true;
         this.server.render('grade').done(
             function(html) {
                 // Load the HTML and install event handlers
                 $(stepID, view.element).replaceWith(html);
                 view.server.renderLatex($(stepID, view.element));
+                view.isRendering = false;
                 view.installHandlers();
                 if (typeof usageID !== 'undefined' &&
                     !$(stepID, view.element).hasClass("is--unfinished") &&
                     !$(stepID, view.element).hasClass("is--unstarted") &&
                     !$(stepID, view.element).hasClass("is--waiting--staff")) {
                     $("[id='oa_grade_" + usageID + "']", view.element).focus();
+                    view.announceStatusChange = true;
+                }
+                if (typeof usageID !== 'undefined' &&
+                    $(stepID, view.element).hasClass("is--showing")){
+                    view.announceStatusChange = true;
```

Comment:
I was avoiding changing existing functionality, there is a specific case on line 37 that has the focus.  I will test this case though to see if it would be valuable to focus here as well.